src/mscorlib/shared/Interop/Windows/Kernel32/Interop.SetLastError.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class Kernel32
+    {
+        [DllImport(Libraries.Kernel32, SetLastError = true)]
+        internal static extern void SetLastError(int lastError);
```

Comment:
SetLastError is not necessary and it does not even work well in CoreCLR. PInvoke marshalling calls will call SetLastError(0) for methods marked with SetLastError=true in CoreCLR.